frontend/src/Utils/Utils.tsx

```diff
@@ -107,6 +109,10 @@ export function validationElim<T, U>(
   }
 }
 
+export function isFree(s : GPUStats): Boolean {
```

Comment:
I prefer `const isFree = (s: GPUStats) => { ... }` style personally. It's not a strong preference, but probably nice to be consistent

Reply:
Also, Utils is ideally intended for very general-purpose utils, not domain-specific stuff comparing gpu usage percentage.